src/app/js/lib/Property.js

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+
+const styles = {
+    container: {
+        display: 'flex',
+        marginRight: '1rem',
+    },
+    name: {
+        fontWeight: 'bold',
+    },
+    scheme: {
+        fontWeight: 'bold',
+        fontSize: '0.75em',
+        color: 'grey',
+    },
+};
+
+const Property = ({ name, value, scheme }) => (
+    <dl className="property" style={styles.container}>
+        <dt>
+            <div style={styles.name}>{name}</div>
+            <div style={styles.scheme}>{scheme}</div>
+        </dt>
+        <dd>{value}</dd>
+    </dl>
+);
+
+Property.propTypes = {
```

Comment:
use (and update) propType defined in [propTypes](https://github.com/Inist-CNRS/lodex-v2/pull/42/files#diff-959dc22805a0b8bc5bacdf8eb672a5f4R11)